src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/health/health.component.ts

```diff
@@ -149,4 +223,13 @@ export class HealthComponent implements OnInit, OnDestroy {
 
     return readOps + writeOps > 0;
   }
+
+  private calcPercentage(dividend: number, divisor: number) {
+    divisor = Number(divisor);
```

Comment:
```suggestion
```

Reply:
@tspmelo Without this line the unit tests do not pass:
What I'm doing here is ensuring that we get either a number or "NaN", otherwise I have to check also if the value received is null (do you prefer the latter option?):
```
Error: expect(received).toEqual(expected)

Expected: 0
Received: Infinity
Expected :0
Actual   :null
 <Click to see difference>


      328 |     it('returns correct value', () => {
      329 |       expect(component['calcPercentage'](1, undefined)).toEqual(0);
    > 330 |       expect(component['calcPercentage'](1, null)).toEqual(0);
          |                                                    ^
```

Reply:
still amazes me how `isNaN` works...
how about using `!_.isNumber` instead of `isNaN`? This is catch `null` values...

Reply:
I agree. Done!